doc/source/programs/gdaldem.rst

```diff
@@ -24,7 +24,7 @@ Generate a shaded relief map:
 .. code-block::
 
      gdaldem hillshade <input_dem> <output_hillshade>
-                 [-z <zfactor>] [-s <scale>]
+                 [-z <zfactor>] [[-s <scale>] | [-xscale <xscale> -xscale <yscale>]]
```

Comment:
`[-xscale <xscale> -xscale <yscale>]]` should  be `[-xscale <xscale> -yscale <yscale>]]`.

Reply:
thanks, fixed